site/content/docs/5.2/components/alerts.md

```diff
@@ -10,6 +10,10 @@ toc: true
 
 Alerts are available for any length of text, as well as an optional close button. For proper styling, use one of the eight **required** contextual classes (e.g., `.alert-success`). For inline dismissal, use the [alerts JavaScript plugin](#dismissing).
 
+{{< callout info >}}
+**Heads up!** As of v5.3.0, the `alert-variant()` Sass mixin is deprecated. Alert variants now have their CSS variables overridden in [the Sass loop](#sass-loop).
```

Comment:
Maybe we could add something like <code>@include deprecate("\`alert-variant()\`", "v5.3.0", "v6.0.0");</code> at the beginning of https://github.com/twbs/bootstrap/blob/f2e15458da24a089356e3de42df8297f6140059e/scss/mixins/_alert.scss#L2